### ISSUE TITLE
[Incompatible api] set_transfinite_surface/volume() 

### ISSUE THREAD

**Issue by BinWang0213** [nschloe/pygmsh#440]
Hi,

In the official gmsh api:
```
def setTransfiniteSurface(tag, arrangement="Left", cornerTags=[])
def setTransfiniteVolume(tag, cornerTags=[])
```
But in pygmsh:
```
def set_transfinite_surface(surface, arrangement, corner_pts):
def set_transfinite_volume(volume, arrangement, corner_pts):
```

So Now I have to use these function as:
```
geom.set_transfinite_surface(s,"Left",[])
geom.set_transfinite_volume(frac_body,None,[])
```

Please update these functions accordingly.

**Comment by nschloe** [comment]
Sounds like an easy PR.

**Comment by nschloe** [comment]
Fixes by #447.
